src/panoptes/pocs/state/machine.py

```diff
@@ -287,12 +305,10 @@ def after_state(self, event_data):
     @classmethod
     def load_state_table(cls, state_table_name='simple_state_table'):
         """ Loads the state table
-
```

Comment:
These are the weird line options that we should figure out for our editors because they seem to be inconsistent.